index/templates/index/index.html

```diff
@@ -12,8 +12,9 @@
             <div class="row">
                 <div class="col-md-6 my-auto text-center">
                     <h2 class="fs-1 fw-bolder">Welcome to United Events</h2>
+                    Your one-stop-shop for major Pride event-related information in the UK this summer<br>&#x1F308;&#x1F31E;&#x1F60E;
```

Comment:
Don't use hexadecimal here `&#x1F308;&#x1F31E;&#x1F60E;`.
Instead just use the emojis: 🌈🌞😎
It's easier for collaborators on the same project to get an idea of what it translates to.

Reply:
I wasn't aware I could&should do it as simply as that.
I'm glad you understood and apparently approved my design layout efforts since they seem to have been deployed very close to my intended aim.